app/server/appsmith-server/src/test/java/com/appsmith/server/solutions/ExamplesOrganizationClonerTests.java

```
@@ -56,14 +53,40 @@
     @Autowired
     private DatasourceService datasourceService;
 
+    @Autowired
+    private OrganizationService organizationService;
+
+    @Autowired
+    private ApplicationPageService applicationPageService;
+
     @Test
     @WithUserDetails(value = "api_user")
-    public void createNewUserValid() {
-        final Mono<Organization> organizationMono = organizationRepository
-                .findByName("Spring Test Organization", AclPermission.READ_ORGANIZATIONS)
+    public void cloneOrganizationWithItsContents() {
+
+        Organization newOrganization = new Organization();
+        newOrganization.setName("Template Organization");
+        final Mono<Organization> requiredDataMono = organizationService.create(newOrganization)
+            .flatMap(organization -> {
+                Application app1 = new Application();
+                app1.setName("1 - public app");
+                app1.setOrganizationId(organization.getId());
+                app1.setIsPublic(true);
+
+                Application app2 = new Application();
+                app2.setOrganizationId(organization.getId());
+                app2.setName("2 - private app");
+
+                return Mono.when(
+                        applicationPageService.createApplication(app1),
+                        applicationPageService.createApplication(app2)
+                ).thenReturn(organization);
+            })
+            .cache();
+
+        final Mono<Organization> organizationMono = requiredDataMono
                 .flatMap(organization -> {
                     if (organization.getId() == null) {
-                        log.error("Cannot find Spring Test Organization");
+                        log.error("Cannot create organization for cloning");
```

This is a test case. Lets throw an error instead of logging this out.